bot.py

```diff
@@ -32,153 +29,59 @@
-        await asyncio.sleep(random.randrange(5, 10))
-        await event.reply(event.message.message)
-
-
-async def send_message_template(
-    user_data, event, start_range, end_range, response_type
-):
-    logging.info(
-        f"Contact: {user_data.contact} - "
-        f"first name: {user_data.first_name} - "
-        f"ID: {user_data.id} - "
-        f"sent message: {event.message.message}"
-    )
-
-    logging.info("Waiting for response...")
-
-
-# TODO send always
-#     async with CLIENT.action(user_data.id, "typing"):
-#         await asyncio.sleep(random.randrange(start_range, end_range))
-#         await CLIENT.send_message(
-#             user_data.id,
-#             f"""
-# Hello, {user_data.first_name}. \n
-# **This message was sent automatically.** \n
-# """,
-#         )
-#         await CLIENT.send_message(user_data.id, response_type)
-#         logging.info(f"Response was sent to {user_data.first_name}.")
-
-
-@CLIENT.on(events.NewMessage(incoming=True, from_users=CIRCULATION_IDS))
-async def response_to_group(event):
-    await show_selected_users()
-
-    user_data = await event.client.get_entity(event.from_id)
+        user_data = await CLIENT.get_entity(db.get_tg_id(current_date))
+        logging.info(
+            f"Today - {current_date} - going to congratulate {user_data.first_name} - {user_data.username}"
+        )
+        await CLIENT.send_message(
+            db.get_tg_id(current_date), random.choice(CONGRATULATIONS_FILE)
+        )
+        async with CLIENT.action(user_data.id, "typing"):
+            await asyncio.sleep(random.randrange(2, 5))
+            await CLIENT.send_file(
+                db.get_tg_id(current_date),
+                bucket.download_file_from_bucket(Path("/home/")),
+            )
+    time.sleep(400)
+    bucket.prune_directory(Path("/home/"))
+    time.sleep(86000)
 
-    logging.info(f"Raw sender data: {user_data}")
 
-    try:
-        if user_data.id in CIRCULATION_IDS:
-            await send_message_template(user_data, event, 5, 10, CIRCULATION_RESPONSE)
-        elif not user_data.contact:
-            logging.info("Looks like someone unfamiliar is on the line.")
-            logging.info(
-                f"Contact: {user_data.contact} - "
-                f"first name: {user_data.first_name} - "
-                f"ID: {user_data.id} - "
-                f"sent message: {event.message.message}"
-            )
-    except ValueError as val_err:
-        logging.error(f"Sender is {user_data.first_name}.")
-        logging.error(val_err)
-    except TypeError as type_err:
-        logging.error("That maybe sticker was sent, not text.")
-        logging.error(f"Sender is {user_data.first_name}.")
-        logging.error(type_err)
-    except BaseException as base_exception:
-        logging.error(f"Sender is {user_data.first_name}.")
-        logging.error(base_exception)
+async def main():
+    while True:
+        await send_congratulations()
```

Comment:
Why don't you put your sleeps here?..

Reply:
It is possible but I prefer to solve this via cloud based solution as I mentioned